spec/controllers/v2/reports_controller_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe V2::ReportsController do
+  before do
+    stub_rbac_permissions(
+      Rbac::INVENTORY_HOSTS_READ,
+      Rbac::SYSTEM_READ,
+      Rbac::REPORT_READ
+    )
+  end
+
+  let(:attributes) do
+    {
+      title: :title,
+      os_major_version: :os_major_version,
+      ref_id: :ref_id,
+      description: :description,
+      profile_title: :profile_title,
+      business_objective: :business_objective,
+      system_count: -> { 0 }, # TODO
```

Comment:
Obviously this can't be zero, if you don't have any systems in the policy, it can't be a report...

Reply:
That's why it says `# TODO` :wink: